### PR TITLE
Adds thirst (and piss)

### PR DESCRIPTION
**Pull request by NullDagaf**
## About The Pull Request

Resprites the hunger alert, adds thirst, adds piss.

## Why It's Good For The Game

Incentivises roleplay by making your characters ACTUALLY use the bathroom, without allowing for the monkey-ing around that adding shit would.

:cl:
add: adds thirst and piss
/:cl:

### CODE REVIEW

**Comment by kaylexis** [comment]
i dont know what to say

**Comment by ShamanSliph** [comment]
How do I delete someone elses branch?

File: modular_zubbers/code/modules/clothing/head/wig.dm [hunk 1]
```diff
@@ -1,3 +1,12 @@
+#define HEAD_ACCESSORIES_PATHS list(\
+	/obj/item/clothing/head,\
+	/obj/item/clothing/glasses,\
+	/obj/item/reagent_containers/cup/bucket,\
+	/obj/item/paper,\
+	/obj/item/nullrod/fedora,\
+)
```
Comment (Cyprex) [inline modular_zubbers/code/modules/clothing/head/wig.dm:7]: Atomize this, if anything.

**Comment by Cyprex** [comment]
We are not interested in this.

**Comment by Cyprex** [comment]
There are numerous technical issues with this PR that would need to be addressed in a review, but that's besides the point- we are simply not interested in a system that makes your character... need to pee periodically.

**Pull request closed without merging**